Julia tests failing

On an obviously [unrelated commit](https://github.com/jayvdb/docker-coala-base/commit/3f1db2f0cf8) the [build](https://travis-ci.org/jayvdb/docker-coala-base/builds/177952132) fails

```
_____________ JuliaRequirementTestCase.test_installed_requirement ______________
self = <JuliaRequirementTest.JuliaRequirementTestCase testMethod=test_installed_requirement>
    def test_installed_requirement(self):
>       self.assertTrue(JuliaRequirement('Lint').is_installed())
E       AssertionError: False is not true
tests/bears/requirements/JuliaRequirementTest.py:11: AssertionError
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -38,6 +38,7 @@ RUN zypper --no-gpg-checks --non-interactive dist-upgrade && \
   libcurl-devel \
   libncurses5 \
   libopenssl-devel \
+  libpcre2-8-0 \
   libxml2-tools \
   lua \
   lua-devel \
@@ -126,7 +127,7 @@ RUN source /etc/profile.d/go.sh \
 # ENV PATH=$PATH:/home/opam/infer-linux64-v0.9.0/infer/bin
 
 # Julia setup
-# RUN ["julia", "-e", "\\"Pkg.add(\\\\"Lint\\\\")\\""]
+RUN julia -e 'Pkg.add("Lint")'
 
 # Lua commands
 RUN luarocks install luacheck

